migrationmanager/migrators/logs/migrations/000014_new_schema.up.sql

```diff
@@ -0,0 +1,119 @@
+CREATE TABLE IF NOT EXISTS  signoz_logs.logs_v2_resource_bucket ON CLUSTER {{.SIGNOZ_CLUSTER}}
+(
+    `labels` String CODEC(ZSTD(5)),
+    `fingerprint` String CODEC(ZSTD(1)),
+    `seen_at_ts_bucket_start` Int64 CODEC(Delta(8), ZSTD(1)),
+    INDEX idx_labels lower(labels) TYPE ngrambf_v1(4, 1024, 3, 0) GRANULARITY 1
+)
+ENGINE = ReplacingMergeTree
+PARTITION BY toDate(seen_at_ts_bucket_start / 1000)
+ORDER BY (labels, fingerprint, seen_at_ts_bucket_start)
+SETTINGS ttl_only_drop_parts = 1, index_granularity = 8192;
+
+
+CREATE TABLE IF NOT EXISTS  signoz_logs.distributed_logs_v2_resource_bucket ON CLUSTER {{.SIGNOZ_CLUSTER}}
+(
+    `labels` String CODEC(ZSTD(5)),
+    `fingerprint` String CODEC(ZSTD(1)),
+    `seen_at_ts_bucket_start` Int64 CODEC(Delta(8), ZSTD(1))
+)
+ENGINE = Distributed('cluster', 'signoz_logs', 'logs_v2_resource_bucket', cityHash64(labels, fingerprint, seen_at_ts_bucket_start));
+
+
+CREATE TABLE IF NOT EXISTS  signoz_logs.logs_v2 ON CLUSTER {{.SIGNOZ_CLUSTER}}
+(
+    `ts_bucket_start` UInt64 CODEC(DoubleDelta, LZ4),
+    `resource_fingerprint` String CODEC(ZSTD(1)),
+    `timestamp` UInt64 CODEC(DoubleDelta, LZ4),
+    `observed_timestamp` UInt64 CODEC(DoubleDelta, LZ4),
+    `id` String CODEC(ZSTD(1)),
+    `trace_id` String CODEC(ZSTD(1)),
+    `span_id` String CODEC(ZSTD(1)),
+    `trace_flags` UInt32,
+    `severity_text` LowCardinality(String) CODEC(ZSTD(1)),
+    `severity_number` UInt8,
+    `body` String CODEC(ZSTD(2)),
+    `attributes_string` Map(String, String) CODEC(ZSTD(1)),
+    `attributes_number` Map(String, Float64) CODEC(ZSTD(1)),
+    `attributes_bool` Map(String, Bool) CODEC(ZSTD(1)),
+    `resources_string` Map(String, String) CODEC(ZSTD(1)),
```

Comment:
Did we not want to use LowCarinality String for keys?

Reply:
we should

Reply:
How many examples have we seen? We should consider the general pattern and use lowcardinality. The special case users can be updated to use regular string if needed but not the other way around.

Reply:
Not many users, yeah it will be better to tell them to reduce the attributes or if required then manually update for them.